### PR TITLE
BUGFIX: Fix static compilation of ThumbnailGeneratorStrategy

### PR DESCRIPTION
**Pull request by bwaidelich**
This fixes `ThumbnailGeneratorStrategy::getThumbnailGeneratorClassNames()` making the
method `public`.
Without this fix, the method is silently skipped from static compilation and will always
use reflection to determine classes implementing `ThumbnailGeneratorInterface`.

Note: With resolution of neos/flow-development-collection#662 this will prevent compilation
in Production context

### CODE REVIEW

**Comment by aertmann** [comment]
but it's not made final as the description say?

**Comment by bwaidelich** [comment]
> but it's not made final as the description say?
fixed ;)

**Comment by bwaidelich** [comment]
Fixed it in the original commit now, too (force-push) and moved keywords & annotations according to our CGL

File: TYPO3.Media/Classes/TYPO3/Media/Domain/Strategy/ThumbnailGeneratorStrategy.php [hunk 1]
```diff
@@ -54,11 +54,11 @@ public function refresh(Thumbnail $thumbnail)
     /**
      * Returns all class names implementing the ThumbnailGeneratorInterface.
      *
+     * @Flow\CompileStatic
      * @param ObjectManagerInterface $objectManager
      * @return ThumbnailGeneratorInterface[]
-     * @Flow\CompileStatic
      */
-    protected static function getThumbnailGeneratorClassNames($objectManager)
+    static public function getThumbnailGeneratorClassNames($objectManager)
```
Comment (aertmann) [inline TYPO3.Media/Classes/TYPO3/Media/Domain/Strategy/ThumbnailGeneratorStrategy.php]: wrong order according to styleci..

Comment (bwaidelich) [inline TYPO3.Media/Classes/TYPO3/Media/Domain/Strategy/ThumbnailGeneratorStrategy.php]: OK, it is indeed the other way around according to PSR-2, fixed it

**Pull request merged**